src/components/utils/src/signals_win.cc

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Please, return *signal_handlers* as a *signals_posix*. We are still expect to merge it back ;)